### PR TITLE
tell biome to read from the version specified in the root directory

### PR DESCRIPTION
**Pull request by salieflewis**

### CODE REVIEW

**Comment by salieflewis** [comment]
Resolves #615

File: .github/workflows/integrate.yml [hunk 1]
```diff
@@ -18,11 +18,6 @@ jobs:
 
     - name: Setup Biome
       uses: biomejs/setup-biome@v2
-      with:
```
Comment (salieflewis) [inline .github/workflows/integrate.yml:21]: The key was deleting this bit in combination with making sure biome was installed at a pinned version, without a caret (`^`)

File: apps/site/biome.json [hunk 2]
```diff
@@ -34,10 +34,5 @@
         "noSvgWithoutTitle": "off"
       }
     }
-  },
```
Comment (salieflewis) [inline apps/site/biome.json:37]: Believe this bit is just unnecessary here.

**Review by 0xTranqui — APPROVED** [review]
looks good to me and approving. only qs i have for u are
- how do we know what version of biome we are happy with / when it calls for an update?
- are there any changes to the workflow of running pnpm format + pnpm lint in the appropriate app folder before pushing up a pr?

**Comment by salieflewis** [comment]
> looks good to me and approving. only qs i have for u are
> 
> * how do we know what version of biome we are happy with / when it calls for an update?
> * are there any changes to the workflow of running pnpm format + pnpm lint in the appropriate app folder before pushing up a pr?

Basically before, biome wasn't able to resolve what the current version installed was because it was looking in the wrong place. By default it pulls from the latest stable version on npm. Thats why in order to fix things we needed to bump biome. This PR fixes that, so whatever version we have installed now will be what it listens to. I think we only need to consider bumping again when a major update occurs and new/wanted functionality becomes available.

There are no changes, and you can run those commands from the root of the repo.

**Pull request merged**